run-e2e.sh

```diff
@@ -32,23 +32,11 @@ curl \
   -o ${WORKSPACE}/repo-list.yaml
 export KUBE_TEST_REPO_LIST=${WORKSPACE}/repo-list.yaml
 
-# Download the list of tests to exclude.
-curl https://raw.githubusercontent.com/e2e-win/e2e-win-prow-deployment/master/exclude_conformance_test.txt -o ${WORKSPACE}/exclude_conformance_test.txt
-# Ignore lines starting with "#"
-EXCLUDED_TESTS=$(cat ${WORKSPACE}/exclude_conformance_test.txt |
-  tr -d '\r' |                # remove Windows carriage returns
-  grep -v '^#' |              # ignore comment lines
-  grep -v '^$' |              # ignore blank lines
-  tr -s '\n' '|' |            # coalesce newlines into |
-  tr -s ' ' '.' |             # coalesce spaces into .
-  sed -e 's/[]\[()]/\\&/g' |  # escape brackets and parentheses
-  sed -e 's/.$//g')           # remove final | added by tr
```

Comment:
But I worked so hard on this command!

Reply:
Haha It will forever live in the git history!